### ISSUE TITLE
add .currentTime() to soundFile

### ISSUE THREAD

**Issue by ertdfgcvb** [processing/processing-sound-archive#53]
It would be an useful addition in many scenarios to get the current time of a playing sound…

**Comment by hamoid** [comment]
Without it it's not possible to know if a sound is done playing, right?

**Comment by ertdfgcvb** [comment]
@hamoid Among other things… But I think it's useful to know the time of a sound if you need to synchronize events to it.

**Comment by wirsing** [comment]
yes, that's true. It is on the list and I hope to have that ready for the next version.

**Comment by b2renger** [comment]
about synchronizing events to sound, it's probably a good idea to mimic the way it's implemented in p5.sound from p5.js.

Check this commit : https://github.com/processing/p5.js-sound/commit/3f71436aa7db8385bf800b0d1bce3fd2dabe92a7#diff-635fd3ddb6f066be99025f1994db3e30R1385

in soundfile.js from line 1385, there are some event scheduling methods, it could be nice to have something like them. Probably not a priority though.

**Comment by dhowe** [comment]
+1 for this -- current time (or frameNumber) seems essential for many use-cases
